### ISSUE TITLE
GoogleMusic plugin menu items not shown

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#109]
```
What steps will reproduce the problem?
1. Properly install the GoogleMusic plugin on your home Logitech Media Server (https://github.com/hechtus/squeezebox-googlemusic).
2. On Squeezer, select "My apps", then "Google Music".

What is the expected output? What do you see instead?
I expect to see the GoogleMusic plugin's menu items ("My Music", "All-Inclusive").
Instead I see no menu items at all. It is not possible to control the GoogleMusic plugin
with Squeezer.

When I use the Logitech Media Server's web interface, I can browse the GoogleMusic
plugin menu items and control the plugin. I have noticed that there are no icons on
the menu items that are missing, so maybe that is the problem?

What version of the product are you using? On what operating system?
Latest version of Squeezer, SqueezeBox server and GoogleMusic plugin.
```

Original issue reported on code.google.com by `shredzone` on 2014-11-16 10:52:34

**Comment by kaaholst** [comment]
```
Added screenshots (in German, sorry).

server.png: The menu on the Logitech Media Server that I am missing on Squeezer.

menu1.png: The "own applications" menu on Squeezer. I click on "Google Music".

menu2.png: This is what I see now. What I expected was the menu that was seen on server.png.
```

Original issue reported on code.google.com by `shredzone` on 2014-11-16 11:17:14

<hr>
- _Attachment: server.png<br>![server.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-83/comment-1/server.png)_
- _Attachment: menu1.png<br>![menu1.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-83/comment-1/menu1.png)_
- _Attachment: menu2.png<br>![menu2.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-83/comment-1/menu2.png)_

**Comment by kaaholst** [comment]
```
I also have this same issue, but have also tried google music through another app (or
rather automation system) that works for everything else, but does the same thing that
I get from Squeezer, no menu items under Google Music.
```

Original issue reported on code.google.com by `dustinww` on 2015-01-26 13:05:56

**Comment by kaaholst** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer


Original issue reported on github.com/nikclayton/android-squeezer by nikclayton on 2015-06-01T11:33:54Z

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:27:00Z
